.github/workflows/reporting-server-migration.yml

```diff
@@ -0,0 +1,57 @@
+name: Reporting Server Migration tests
+on:
+  pull_request:
+    paths:
+      - 'packages/reporting-server/**'
+  push:
+    branches:
+      - main
+jobs:
+  # Label of the container job
+  migrations:
+    # Containers must run in Linux based operating systems
+    runs-on: ubuntu-latest
+    # Service containers to run with `container-job`
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres:latest
+        ports:
+          - 5432:5432
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_USER: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      # Downloads a copy of the code in your repository before running CI tests
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install python dependencies
+        run: pip3 install tortoise-orm pydantic aerich asyncpg 
```

Comment:
Yes, but correct me if I'm wrong, I think we need to have the whl to install the dependencies. So I need to build the project first. To do that I need from `lerna` and `node`. So maybe I can use the reporting-server docker image on the docker registry to get dependencies I need?

Reply:
Yeah, as a alternative, we can `pipenv install`, but since we share the same `Pipfile` for all the projects, this would install other unneeded dependencies as well.

If reporting-server is "pip3 installable", then we can install it from source as "editable" e.g. `pip3 install .`, this should bring in only the needed runtime dependencies.